src/SprykerEco/Zed/UnzerApi/Business/Api/Response/UnzerApiResponseConstants.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Response;
+
+interface UnzerApiResponseConstants
+{
+    /**
+     * @var string
+     */
+    public const PARAM_CODE = 'code';
+    /**
+     * @var string
+     */
+    public const PARAM_ERRORS = 'errors';
+    /**
+     * @var string
+     */
+    public const PARAM_TIMESTAMP = 'timestamp';
+    /**
+     * @var string
+     */
+    public const PARAM_URL = 'url';
+    /**
+     * @var string
+     */
+    public const PARAM_MERCHANT_MESSAGE = 'merchantMessage';
+    /**
+     * @var string
+     */
+    public const PARAM_CUSTOMER_MESSAGE = 'customerMessage';
+}
```

Comment:
empty lines between would be great ;)

Reply:
Empty lines between consts are not mandatory